target/linux/realtek/dts-5.10/rtl930x.dtsi

```diff
@@ -8,14 +10,57 @@
 
 	compatible = "realtek,rtl838x-soc";
 
+	ccu: clock-controller {
+		compatible = "realtek,rtl9300-clock";
+		#clock-cells = <1>;
+	};
+
 	cpus {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		frequency = <800000000>;
 
 		cpu@0 {
 			compatible = "mips,mips34Kc";
 			reg = <0>;
+			clocks = <&ccu CLK_CPU>;
+			operating-points-v2 = <&cpu_opp_table>;
+		};
+
+		cpu@1 {
```

Comment:
Same comment as for the sram-scratch nodes. If this is still unused at this point, please leave it out of the change set. AFAICT, adding the second VPE is not required for switching over to the new clock driver; which this commit is supposedly doing.

Reply:
removed